doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -4096,7 +4094,7 @@ which the function is supplied:
    This tactic:
 
    + checks that the goal is a Leibniz equality;
-   + matches both sides of this equality with “term applied to some arguments”, inferring the right number of arguments from the goal and the type of term. This may expand some definitions or fixpoints;
+   + matches both sides of this equality with “term applied to some arguments”, inferring the right number of arguments from the goal and the type of term (this may expand some definitions or fixpoints);
```

Comment:
```suggestion
   + matches both sides of this equality with “term applied to some arguments”,
     inferring the right number of arguments from the goal and the type of ``term``
     (this may expand some definitions or fixpoints);
```